google.analytics/ga.d.ts

```diff
@@ -38,13 +38,46 @@ interface GoogleAnalytics {
 
 declare module UniversalAnalytics {
     // https://developers.google.com/analytics/devguides/collection/analyticsjs/method-reference
+    
+    enum HitType {
+        'pageview', 'screenview', 'event', 'transaction', 'item', 'social', 'exception', 'timing'
+    }
 
     interface ga {
         l: number;
         q: any[];
+
+        (command: 'send', hitType: 'event', eventCategory: string, eventAction: string,
+            eventLabel?: string, eventValue?: number, fieldsObject?: {}): void;
+        (command: 'send', hitType: 'event', fieldsObject: {
+            eventCategory: string,
+            eventAction: string,
+            eventLabel?: string,
+            eventValue?: number,
+            nonInteraction?: boolean}): void;
+        (command: 'send', fieldsObject: {
+            hitType: HitType, // 'event'
+            eventCategory: string,
+            eventAction: string,
+            eventLabel?: string,
+            eventValue?: number,
+            nonInteraction?: boolean}): void;
+        (command: 'send', hitType: 'pageview', page: string): void;
+        (command: 'send', hitType: 'social',
+            socialNetwork: string, socialAction: string, socialTarget: string): void;
+        (command: 'send', hitType: 'social',
+            fieldsObject: {socialNetwork: string, socialAction: string, socialTarget: string}): void;
+        (command: 'send', hitType: 'timing',
+            timingCategory: string, timingVar: string, timingValue: number): void;
+        (command: 'send', hitType: 'timing',
+            fieldsObject: {timingCategory: string, timingVar: string, timingValue: number}): void;
+        (command: 'send', hitType: HitType, ...fields: any[]): void;
+        (command: 'send', fieldsObject: {}): void;
+        
```

Comment:
`(command: string, hitType: string, ...fields: any[]): void;`
We need this line for non-specialized signature.